R/set.R

```diff
@@ -184,15 +204,9 @@ m_set_preceived_distance <- function(id, dist) {
 
 #' @rdname m_set_view
 #' @export
-m_set_view <- function(id, arg) {
+m_set_view <- function(id, arg, style) {
   method <- "setView"
   callJS()
 }
 
 
-#' @rdname m_set_view
-#' @export
-m_set_view_style <- function(id, style) {
-  method <- "setViewStyle"
-  callJS()
-}
```

Comment:
It's more safe that we first add some warning message in this function, and tell the user we will delete it in next version, instead of abruptly deleting it.

Reply:
I deleted this function and replaced it with the `m_add_outline()` function. From what I could tell from diving into the javascript function itself, the only option that was possible was changing the outline or not. 

I'll add it back in with a warning. If you can figure out if it does anything else, let me know

Reply:
Yes, you are absolutely right. I suspect that there are very few users of the package at the moment, and deleting it directly won't cause much trouble. But it would be better to consider deleting it one version apart just in case. 👍🏼